sanity-suite/src/testBookSuites/eventFiltering.js

```diff
@@ -0,0 +1,104 @@
+import eventFiltering1ExpectedData from '../../__mocks__/eventFiltering1.json';
+
+const eventFilteringSuite = {
+  id: 'eventFilteringFeature',
+  name: 'eventFiltering',
+  description:
+    'Track Method: rudderanalytics.track(event, [properties], [apiOptions], [callback]);',
+  testCases: [
+    {
+      id: 'eventFiltering',
+      description: 'Call with all arguments and all primitives in properties and apiOptions',
+      inputData: [
```

Comment:
This is not the right payload

Reply:
the format should be something like below:
```
inputData:[
"event name here",
{
  key: value // properties
  order_id: "sample order id",
  products:[]
} ,
{
  anotherKey: anotherValue
}
]
```

Reply:
What you have right now is the SDK-generated payload.

Reply:
@MoumitaM  Pushed the new changes, can you please verify